crates/change_json/src/change_json.rs

```diff
@@ -64,6 +65,23 @@ impl From<Change> for ChangeJson {
     }
 }
 
+pub trait Find {
+    fn find_crate(&self, display_name: &str) -> Option<CrateId>;
+}
+
+impl Find for CrateGraph {
+    fn find_crate(&self, display_name: &str) -> Option<CrateId> {
```

Comment:
is the `impl` for `CrateGraph` used somewhere? I could only find the one for `Change`.

Reply:
Yes, in the implementation for Change ;)